com.unity.render-pipelines.universal/Shaders/ShadowCasterPass.hlsl

```diff
@@ -25,7 +29,13 @@ float4 GetShadowPositionHClip(Attributes input)
     float3 positionWS = TransformObjectToWorld(input.positionOS.xyz);
     float3 normalWS = TransformObjectToWorldNormal(input.normalOS);
 
-    float4 positionCS = TransformWorldToHClip(ApplyShadowBias(positionWS, normalWS, _LightDirection));
+#if _CASTING_PUNCTUAL_LIGHT_SHADOW
+    float3 lightDirectionWS = normalize(_LightPosition - positionWS);
+#else
+    float3 lightDirectionWS = _LightDirection;
+#endif
+
+    float4 positionCS = TransformWorldToHClip(ApplyShadowBias(positionWS, normalWS, lightDirectionWS));
```

Comment:
please make sure that any changes to the passes here are also reflected in the ShaderGraph includes, in Editor/Shader Graph/Includes

Reply:
Thanks for the comment, I updated the ShaderGraph headers to include this change as well (revision 05ea77e).

I verified that replacing the materials used in test scene TestProjects/UniversalGraphicsTest/Assets/Scenes/145_ManyAdditionalLightShadowTypes.unity by materials using ShaderGraph shaders, produced the same visual results.

I also verified that the changes in this pull request branch did not change the rendering of the scene in https://github.com/Unity-Technologies/ShaderGraph-Custom-Lighting